dev/integration_tests/android_semantics_testing/test_driver/main_test.dart

```diff
@@ -120,29 +120,7 @@ void main() {
         // Ideally this should test the case where there is nothing on the
         // clipboard as well, but there is no reliable way to clear the
         // clipboard on Android devices.
-        final SerializableFinder normalTextField = find.descendant(
```

Comment:
I can't really tell why this didn't work some of the time, it looks like running this setup with some state leaked from other test will cause it to fail.
I think it is ok to bypass this setup since this is not the focus of this test